components/installer/pkg/steps/dex.go

```diff
@@ -10,13 +10,24 @@ import (
 )
 
 //InstallDex installs Dex component
-func (steps *InstallationSteps) InstallDex(installationData *config.InstallationData) error {
+func (steps *InstallationSteps) InstallDex(installationData *config.InstallationData, overrideData OverrideData) error {
 
 	const stepName string = "Installing Dex"
 	const namespace string = "kyma-system"
 
 	chartDir := path.Join(steps.chartDir, consts.DexComponent)
-	overrides, err := steps.getDexOverrides(installationData, chartDir)
+	dexOverrides, err := steps.getDexOverrides(installationData, chartDir)
+
+	if steps.errorHandlers.CheckError("Install Overrides Error: ", err) {
+		steps.statusManager.Error(stepName)
+		return err
+	}
+
+	allOverrides := overrides.Map{}
+	overrides.MergeMaps(allOverrides, overrideData.Common())
+	overrides.MergeMaps(allOverrides, dexOverrides) //TODO: Remove after migration to generic overrides is completed
+	overrides.MergeMaps(allOverrides, overrideData.ForComponent(consts.DexComponent))
+	overridesStr, err := overrides.ToYaml(allOverrides)
```

Comment:
Once the migration is over, it might be a good idea to provide a generic function applicable to all components.
getAllOverrides(stepName string) (string, err) {...}